# generated by patch-package 6.4.8 on 2022-02-18 22:50:57
#
# command:
#   npx patch-package vite-plugin-tauri
#
# declared package:
#   vite-plugin-tauri: 1.1.0
#
diff --git a/node_modules/vite-plugin-tauri/bin/vite-tauri.js b/node_modules/vite-plugin-tauri/bin/vite-tauri.js
index dd4e5d7..c94d428 100755
--- a/node_modules/vite-plugin-tauri/bin/vite-tauri.js
+++ b/node_modules/vite-plugin-tauri/bin/vite-tauri.js
@@ -36,7 +36,7 @@ import { relative, dirname } from "path";
 
 // src/init.ts
 import enquirer from "enquirer";
-import TauriCli from "@tauri-apps/cli/main";
+import TauriCli from "@tauri-apps/cli/main.js";
 async function init(args, promptUser = false) {
   if (promptUser) {
     const { confirmed } = await enquirer.prompt({
